src/store/metadata/stub/content/editions/versions/gdhi_2023-03.json

```diff
@@ -12,72 +12,72 @@
             "@id": "https://staging.idpd.uk/datasets/gdhi/editions/2023-03/versions/1",
             "identifier": "1",
             "issued": "2023-03-31T12:12:48Z",
-            "title": "Consumer Prices Index including owner occupiers' housing costs (CPIH)",
+            "title": "Gross disposable household income per head",
             "summary": "Estimate of the amount of money each individual within a household has available for spending or saving after they have paid taxes and received any direct benefits, UK, 1997-2021.",
             "description": "This dataset shows an estimate of the amount of money each individual within a household has available for spending or saving after they have paid taxes and received any direct benefits, in the UK, for 1997-2021.",
             "download_url": "https://staging.idpd.uk/datasets/gdhi/editions/2023-03/versions/1.csv",
             "media_type": "text/csv",
             "table_schema": {
                 "columns": [
                     {
-                        "name": "Period",
+                        "name": "period",
                         "datatype": "string",
-                        "titles": "Period",
+                        "titles": "Year",
                         "description": "The time period associated with the observation."
                     },
                     {
-                        "name": "AREANM",
+                        "name": "areanm",
                         "datatype": "string",
                         "titles": "Area Name",
-                        "description": "The name assiciated with The geography code "
+                        "description": "The name assiciated with the geography code "
                     },
                     {
-                        "name": "AREACD",
+                        "name": "areacd",
                         "datatype": "string",
-                        "titles": "Statistical Geography",
+                        "titles": "ONS Geography Code",
                         "description": "The geography code associated with the observation."
                     },
                     {
-                        "name": "Geography",
+                        "name": "geography",
                         "datatype": "string",
                         "titles": "Geography Type",
                         "description": "The categorisation of the geography code"
                     },
                     {
-                        "name": "Value",
+                        "name": "observation",
                         "datatype": "decimal",
-                        "titles": "Geographic area with 4G coverage by at least one provider",
-                        "description": "Observational Value"
+                        "titles": "Gross disposable household income per head",
+                        "description": "Measure Type of the Observational Value"
```

Comment:
Measure type is laid out below. Description needs to reflect what the observation is.